### DISCUSSION TITLE
Backticks in link titles

### DISCUSSION THREAD

**Discussion by aremes** [mkdocs/mkdocs#2849] [Q&A]
The following markdown code
```[one](1 'first`')[two](2 'second`')```
renders as a single link with a title that contains ```first`')[two](2 'second```
or in html: ```<a href="1" title="first')[two](2 'second">one</a>```

I have found no other markdown-viewer online that does this. Here's how github (correctly) displays the code from above:

[one](1 'first`')[two](2 'second`')

(two links with titles containing a backtick)

Is there any way to configure mkdocs to behave that way?

Background: I'm trying to have Markdown files generated from .NET xml documentation converted to html documentation using MkDocs. Because in .NET, generic type arguments are represented by backticks suffixed to class name, all generic types within the API that i'm trying to document produce broken links.

**Comment by oprypin** [comment]
Markdown within MkDocs is handled almost fully by [Python-Markdown](https://github.com/Python-Markdown/markdown) - that's where you'd look to try to get different behavior.

```pycon
>>> import markdown
>>> print(markdown.markdown("""[one](1 'first`')[two](2 'second`')"""))
<p><a href="1" title="first')[two](2 'second">one</a></p>
```

**Reply by aremes** [reply]
Thanks! I havent found anything but hope this can be done using an extension, maybe.. we'll see
